internal/backends/winit/renderer/femtovg/fonts.rs

```diff
@@ -739,5 +739,5 @@ pub(crate) fn layout_text_lines(
             start = index;
         }
     }
-    baseline_y
+    y
```

Comment:
Good idea. The call site I think needs fixing as well, just for code reading purposes. It currently says

```rust
let baseline_y = fonts::layout_text_lines(
```

but should probably be

```rust
let cursor_y_at_end_of_text = fonts::layout_text_lines(
```